packages/vat-data/src/kind-utils.js

```diff
@@ -0,0 +1,69 @@
+import { provide } from '@agoric/store';
+import { defineDurableKind, makeKindHandle } from './vat-data-bindings.js';
+
+/** @template L,R @typedef {import('@endo/eventual-send').RemotableBrand<L, R>} RemotableBrand */
+
+export const dropContext =
+  fn =>
+  (_, ...args) =>
+    fn(...args);
+// @ts-expect-error TODO statically recognize harden
+harden(dropContext);
+
+export const provideKindHandle = (baggage, kindName) =>
+  provide(baggage, `${kindName}_kindHandle`, () => makeKindHandle(kindName));
+// @ts-expect-error TODO statically recognize harden
+harden(provideKindHandle);
+
+/**
+ * By analogy with how `Array.prototype.map` will map the elements of
+ * an array to transformed elements of an array of the same shape,
+ * `objectMap` will do likewise for the string-named own enumerable
+ * properties of an object.
+ *
+ * See the comment at
+ * https://github.com/Agoric/agoric-sdk/pull/5674#discussion_r908883510
+ * TODO need to move this to a common place.
+ *
+ * @template T, U
+ * @template {keyof T} K
+ * @param {{ [K2 in keyof T]: T[K2] }} original
+ * @param {(pair: [K, T[K]]) => [K, U]} mapPairFn
+ * @returns {Record<K, U>}
+ */
+export const objectMap = (original, mapPairFn) => {
```

Comment:
this abstraction doesn't seem right to me. Like `map` of Array, I would expect it to map the value to another value. This leaks the `Object.entries` tuple concept.

the one place this is used would be better served by a cleaner abstraction:
```
const behavior = objectMap(methods, dropContext);
```

Reply:
Done. Also consolidated the two implementations of `objectMap` into one and corrected the comment. Much improved!